### PR TITLE
Revert "Default to classic queues v2"

### PR DESCRIPTION
**Pull request by lhoguin**
This reverts commit a404b49d763e9bc0f8681546de87a7aed884f8d9 from https://github.com/rabbitmq/rabbitmq-server/pull/8308

The second commit is not reverted as it is just a test change that is useful regardless of the default change.

### CODE REVIEW

File: release-notes/3.13.0.md [hunk 1]
```diff
@@ -240,21 +240,6 @@ Release notes are kept under [rabbitmq-server/release-notes](https://github.com/
...
-
-   ``` ini
-   # uses CQv1 by default
-   classic_queue.default_version = 1
-   ```
-
-   Individual queues can be declared by passing `x-queue-version` argument and/or through a `queue-version` policy.
-
-   GitHub issue: [#8308](https://github.com/rabbitmq/rabbitmq-server/pull/8308)
-
```
Comment (lhoguin) [inline release-notes/3.13.0.md:257]: Maybe this should be replaced with a note about deprecation of CQv1 and recommendation to update queues to v2 using policies before v1 gets removed in 4.0.

Comment (michaelklishin) [reply release-notes/3.13.0.md:257]: Yes, definitely, let me edit the release notes.

**Comment by lhoguin** [comment]
All green.

**Comment by michaelklishin** [comment]
@lhoguin I have updated the release notes, at least for now this should be sufficient. Next up is updating the 3.13 blog post.

**Pull request merged**